python/makeSummarySMSplots.py

```diff
@@ -0,0 +1,252 @@
+        LObs.SetPoint(1,self.Xmin+77*xRange/100, self.Ymax-1.35*yRange/10+offset)
+
+        textObs = rt.TLatex(self.Xmin+78*xRange/100, self.Ymax-1.50*yRange/10+offset, "Expected")
+        textObs.SetTextFont(42)
+        textObs.SetTextSize(0.030)
+        textObs.Draw()
+        textObsList.append(textObs)
+
+        offset = -(iM+0.80)*(yRange/18.)
+        LExp= rt.TGraph(2)
+        LExp.SetName("LExp")
+        LExp.SetTitle("LExp")
+        LExp.SetLineColor(rt.kBlack)
+        LExp.SetLineStyle(1)
+        LExp.SetLineWidth(4)
+        LExp.SetMarkerStyle(20)
+        LExp.SetPoint(0,self.Xmin+70*xRange/100, self.Ymax-1.35*yRange/10+offset)
+        LExp.SetPoint(1,self.Xmin+77*xRange/100, self.Ymax-1.35*yRange/10+offset)
+        textObs = rt.TLatex(self.Xmin+78*xRange/100, self.Ymax-1.50*yRange/10+offset, "Observed")
+        textObs.SetTextFont(42)
+        textObs.SetTextSize(0.030)
+        textObs.Draw()
+        textObsList.append(textObs)
+
+
+        LObs.Draw("LSAME")
+        LExp.Draw("LSAME")
+        LObsList.append(LObs)
+        LExpList.append(LExp)
+
+        self.c.LObsList = LObsList
+        self.c.LExpList = LExpList
+        self.c.textObsList = textObsList
+
+    def DrawText(self):
+        #redraw axes
+        self.contPlotDict[self.modelNames[0]].c.RedrawAxis()
+        # white background
+        graphWhite = rt.TGraph(5)
+        graphWhite.SetName("white")
+        graphWhite.SetTitle("white")
+        graphWhite.SetFillColor(rt.kWhite)
+        graphWhite.SetFillStyle(1001)
+        graphWhite.SetLineColor(rt.kBlack)
+        graphWhite.SetLineStyle(1)
+        graphWhite.SetLineWidth(3)
+        graphWhite.SetPoint(0,self.Xmin, self.Ymax)
+        graphWhite.SetPoint(1,self.Xmax, self.Ymax)
+        graphWhite.SetPoint(2,self.Xmax, self.Ymax*(1-(len(self.contPlotDict)+0.5)/18-1.2/9))
+        graphWhite.SetPoint(3,self.Xmin, self.Ymax*(1-(len(self.contPlotDict)+0.5)/18-1.2/9))
+        graphWhite.SetPoint(4,self.Xmin, self.Ymax)
+        graphWhite.Draw("FSAME")
+        graphWhite.Draw("LSAME")
+        self.c.graphWhite = graphWhite
+       	CMS_lumi.writeExtraText = 0
+	CMS_lumi.extraText = self.preliminary
+	CMS_lumi.lumi_13TeV = self.lumi+" fb^{-1}"
+
+	CMS_lumi.lumi_sqrtS = self.energy+" TeV"  
+	iPos=0
+	CMS_lumi.CMS_lumi(self.c,4, iPos)
+        # CMS LABEL
+        textCMS = rt.TLatex(0.25,0.96,"  %s " %(self.preliminary))
+        textCMS.SetNDC()
+        textCMS.SetTextAlign(13)
+        textCMS.SetTextFont(52)
+        textCMS.SetTextSize(0.038)
+        textCMS.Draw()
+        self.c.textCMS = textCMS
+        # MODEL LABEL
+        textModelLabel= rt.TLatex(0.15,0.90,"%s   NLO+NLL exclusion" %self.label)
+        textModelLabel.SetNDC()
+        textModelLabel.SetTextAlign(13)
+        textModelLabel.SetTextFont(42)
+        textModelLabel.SetTextSize(0.035)
+        textModelLabel.Draw()
+        self.c.textModelLabel = textModelLabel
+        pass
+    def DrawDiagonalMTop(self):
```

Comment:
why you redefined these functions? They are already defined here:
https://github.com/CMSRA1/PlotsSMS/blob/ra1_master/python/smsPlotABS.py#L329